lib/licenses.nix

```
@@ -1,1284 +1,1293 @@
+      free = false;
+    };
+
+    cc-by-nc-sa-20 = {
+      spdxId = "CC-BY-NC-SA-2.0";
+      fullName = "Creative Commons Attribution Non Commercial Share Alike 2.0";
+      free = false;
+    };
+
+    cc-by-nc-sa-25 = {
+      spdxId = "CC-BY-NC-SA-2.5";
+      fullName = "Creative Commons Attribution Non Commercial Share Alike 2.5";
+      free = false;
+    };
+
+    cc-by-nc-sa-30 = {
+      spdxId = "CC-BY-NC-SA-3.0";
+      fullName = "Creative Commons Attribution Non Commercial Share Alike 3.0";
+      free = false;
+    };
+
+    cc-by-nc-sa-40 = {
+      spdxId = "CC-BY-NC-SA-4.0";
+      fullName = "Creative Commons Attribution Non Commercial Share Alike 4.0";
+      free = false;
+    };
+
+    cc-by-nc-30 = {
+      spdxId = "CC-BY-NC-3.0";
+      fullName = "Creative Commons Attribution Non Commercial 3.0 Unported";
+      free = false;
+    };
+
+    cc-by-nc-40 = {
+      spdxId = "CC-BY-NC-4.0";
+      fullName = "Creative Commons Attribution Non Commercial 4.0 International";
+      free = false;
+    };
+
+    cc-by-nd-30 = {
+      spdxId = "CC-BY-ND-3.0";
+      fullName = "Creative Commons Attribution-No Derivative Works v3.00";
+      free = false;
+    };
+
+    cc-by-sa-10 = {
+      spdxId = "CC-BY-SA-1.0";
+      fullName = "Creative Commons Attribution Share Alike 1.0";
+    };
+
+    cc-by-sa-20 = {
+      spdxId = "CC-BY-SA-2.0";
+      fullName = "Creative Commons Attribution Share Alike 2.0";
+    };
+
+    cc-by-sa-25 = {
+      spdxId = "CC-BY-SA-2.5";
+      fullName = "Creative Commons Attribution Share Alike 2.5";
+    };
+
+    cc-by-10 = {
+      spdxId = "CC-BY-1.0";
+      fullName = "Creative Commons Attribution 1.0";
+    };
+
+    cc-by-20 = {
+      spdxId = "CC-BY-2.0";
+      fullName = "Creative Commons Attribution 2.0";
+    };
+
+    cc-by-30 = {
+      spdxId = "CC-BY-3.0";
+      fullName = "Creative Commons Attribution 3.0";
+    };
+
+    cc-by-sa-30 = {
+      spdxId = "CC-BY-SA-3.0";
+      fullName = "Creative Commons Attribution Share Alike 3.0";
+    };
+
+    cc-by-40 = {
+      spdxId = "CC-BY-4.0";
+      fullName = "Creative Commons Attribution 4.0";
+    };
+
+    cc-by-sa-40 = {
+      spdxId = "CC-BY-SA-4.0";
+      fullName = "Creative Commons Attribution Share Alike 4.0";
+    };
+
+    cddl = {
+      spdxId = "CDDL-1.0";
+      fullName = "Common Development and Distribution License 1.0";
+    };
+
+    cecill20 = {
+      spdxId = "CECILL-2.0";
+      fullName = "CeCILL Free Software License Agreement v2.0";
+    };
+
+    cecill21 = {
+      spdxId = "CECILL-2.1";
+      fullName = "CeCILL Free Software License Agreement v2.1";
+    };
+
+    cecill-b = {
+      spdxId = "CECILL-B";
+      fullName = "CeCILL-B Free Software License Agreement";
+    };
+
+    cecill-c = {
+      spdxId = "CECILL-C";
+      fullName = "CeCILL-C Free Software License Agreement";
+    };
+    commons-clause = {
+      fullName = "Commons Clause License";
+      free = false;
+    };
```

```suggestion

    commons-clause = {
      fullName = "Commons Clause License";
      url = "https://commonsclause.com/";
      free = false;
    };

```

and please only change the newly-added lines; the rest of the file doesn't need to be changed.